src/libraries/Common/src/System/HexConverter.cs

```diff
@@ -78,7 +78,7 @@ public static void ToCharsBuffer(byte value, Span<char> buffer, int startingInde
 
         internal static unsafe string ToString(ReadOnlySpan<byte> bytes, Casing casing = Casing.Upper)
         {
-#if NET45 || NET461 || NET462 || NET472 || NETSTANDARD1_0 || NETSTANDARD1_3 || NETSTANDARD2_0
```

Comment:
Can this not just be `#if (NETFRAMEWORK && !NET48) || ...`?

We've said `NET48` is the final .NET Framework version, so it shouldn't need to expand in the future. We likewise might be able to do `NETSTANDARD && !NETSTANDARD2_1`, unless we think there will be another netstandard version in the future.

Reply:
I'd leave expressed this way.

* It shows what all modes are hitting this
* It calls things to attention if we add a NET452 
* It hedges against there one day being a NET481.
  * Yeah, we've said there won't be; but it's not like (NETFRAMEWORK && !NET48) is much smaller.
* It avoids mixing && and ||.